VespaGAS/Managers/State/State.swift

```diff
@@ -0,0 +1,5 @@
+import ReSwift
+
```

Comment:
The application state is defined in a single data structure. The idea is that this struct should store the ENTIRE state of your app 😓